lib/id3c/cli/command/etl/fhir.py

```diff
@@ -844,12 +846,19 @@ def process_address(location: Location, tract_hierarchy: str) -> Optional[Any]:
 
     try:
         parent_location = location.partOf.resolved(Location)
-        tract_hierarchy = get_tract_hierarchy(parent_location)  # TODO do we only want parent hierarchy?
+        tract_hierarchy = get_tract(parent_location).hierarchy  # TODO do we only want parent hierarchy?
 
     except AttributeError:
         LOG.info(f"No parent location found for Location with code «{code}», " + \
             f"relation «{relation}».")
-        tract_hierarchy = None
+        try: # handle encounter locations that are only tracts
+            tract = get_tract(location)
+            tract_hierarchy = tract.hierarchy
+            tract_id = tract.id
+        except:
```

Comment:
We should generally try to avoid blanket `except` statements. Are we trying to catch the `AssertionError` here in the case that the tract is unknown? Maybe we are also trying to catch the `AttributeError` in the case of `tract.hierarchy` not being defined?